bundle-workflow/README.md

```diff
@@ -103,41 +103,52 @@ Tests the OpenSearch bundle.
 
 This workflow contains integration, backwards compatibility and performance tests.
 
-The following example kicks off all test suites for a distribution of OpenSearch 1.1.0.
-
+Usage:
 ```bash
-./bundle-workflow/test.sh manifests/opensearch-1.1.0.yml
+export AWS_ROLE_ARN=arn:aws:iam::<AWS_JENKINS_ACCOUNT>:role/opensearch-test
+export AWS_ROLE_SESSION_NAME=dummy-session
+
+Next, configure temporary credentials in environment w/
+export AWS_SESSION_TOKEN=<value>
+export AWS_ACCESS_KEY_ID=<value>
+export AWS_SECRET_ACCESS_KEY=<value>
```

Comment:
Do we need this if we are using role in line 108?

Reply:
Good question. To assume a role, it needs the right credentials. 
- For Jenkins, its taken care by the instance role.
- For developers on local desktops, they have to export these creds to assume that role.